SwiftGit2/SwiftGit2.m

```diff
@@ -34,3 +34,21 @@ git_checkout_options SG2CheckoutOptions(SG2CheckoutProgressBlock progress) {
 	
 	return result;
 }
+
+git_clone_options SG2CloneOptions() {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-field-initializers"
+	git_clone_options result = GIT_CLONE_OPTIONS_INIT;
+#pragma clang diagnostic pop
+
+	return result;
+}
+
+git_fetch_options SG2FetchOptions() {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-field-initializers"
+	git_fetch_options result = GIT_FETCH_OPTIONS_INIT;
+#pragma clang diagnostic pop
+
+	return result;
```

Comment:
Out of curiousity, why the preference for moving logic into Objective-C rather than Swift?

I initially did follow a similar pattern as CheckoutOptions but I just ended having to pull more and more into Objective-C land, so I changes to moving it into Swift now 2 can handle C function pointers.